packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -65,6 +66,8 @@ const customModuleRegexes = bpkReactScriptsConfig.babelIncludePrefixes
   : [];
 
 const optInCssModules = bpkReactScriptsConfig.cssModules === false;
+const optimizeCssOptions = bpkReactScriptsConfig.optimizeCssOptions ? bpkReactScriptsConfig.optimizeCssOptions : {}
```

Comment:
Whats the thinking here? i.e .why would we need to expose those options to userland config? We don't do this for autoprefixer for instance...